app/models/remote_execution_feature.rb

```diff
@@ -1,7 +1,7 @@
 class RemoteExecutionFeature < ActiveRecord::Base
   attr_accessible :label, :name, :provided_input_names, :description, :job_template_id
 
-  validate :label, :name, :presence => true, :unique => true
+  validates :label, :name, :presence => true, :uniqueness => true
```

Comment:
Aight, I'll keep the unit tests though, as I don't see it anywhere else (they would've prevented this)